source/index.html

```diff
@@ -1,77 +1,125 @@
-<!doctype html>
-<!--[if lt IE 7]>      <html lang="en-us" class="lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html lang="en-us" class="lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html lang="en-us" class="lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html lang="en-us"> <!--<![endif]-->
+<!DOCTYPE html>
+<html>
 
-    <head>
-        <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <title>Panara Imports - Marquette, MI</title>
-        <meta name="description" content="Unique fashions &amp; fabulous finds since 1992">
-        <meta name="viewport" content="width=device-width">
-        <link rel="stylesheet" type="text/css" href="./main.css">
-    </head>
+<head>
+    <title>Panara - Imports</title>
+    <link rel="stylesheet" type="text/css" href="main.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
The head should probably look like this (you can remove the comments and the whitespace):
```html
<!-- these go first -->
<meta charset="utf-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">

<!-- these go together -->
<title>Panara - Imports</title>
<meta name="description" content="Unique fashions &amp; fabulous finds since 1992">

<!-- this (and any other head-specific script tags) can go last -->
<link rel="stylesheet" type="text/css" href="main.css">
```